xbmc/video/PlayerController.h

```diff
@@ -29,11 +30,10 @@
  altering subtitles and audio tracks, changing aspect ratio, subtitle placement,
  and placement of the video on screen.
  */
-class CPlayerController : public ISliderCallback
+class CPlayerController : public ISliderCallback, public IActionListener
```

Comment:
Could you add a "virtual" to the OnAction method - for me this indicates that this is from the interface somehow.